app/Http/Controllers/HomeController.php

```diff
@@ -15,14 +16,25 @@ public function __construct()
     {
         $this->middleware('auth');
     }
-    
     /**
      * Show the application dashboard.
      *
      * @return \Illuminate\Contracts\Support\Renderable
      */
     public function index()
     {
-        return view('home');
+        $blogs = Auth::user()->blogs()->orderBy('created_at', 'INC')->get();
```

Comment:
We do not have blogs in your application, so please remove also.

Reply:
It has been deleted.